src/output-view.js

```diff
@@ -0,0 +1,109 @@
+import {
+    Transformime,
+    TextTransformer,
+    ImageTransformer,
+    HTMLTransformer
+} from "transformime";
+
+import {
+    consoleTextTransform,
+    markdownTransform,
+    LaTeXTransform,
+    PDFTransform
+} from "transformime-jupyter-transformers";
+
+/**
+ * Displays output area state.
+ */
+export class OutputView {
+    
+    /**
+     * Public constructor
+     * @param  {IOutputModel} model    output model that this view represents
+     * @param  {Document}     document handle to the Document instance that 
+     *                                 the output will be rendered in.
+     */
+    constructor(model, document) {
+        this.model = model;
+        
+        let el = document.createElement('div');
+        Object.defineProperty(this, 'document', {get: () => document});
+        Object.defineProperty(this, 'el', {get: () => el});
+        
+        // Transformers are in reverse priority order
+        // so that new ones can be `push`ed on with higher priority
+        let transformers = [
+            TextTransformer,
+            PDFTransform,
+            ImageTransformer,
+            // SVG would go here, IF I HAD ONE
+            consoleTextTransform,
+            LaTeXTransform,
+            markdownTransform,
+            HTMLTransformer
+            // JavaScript would go here, IF I HAD ONE
+        ];
+        this.transformime = new Transformime(transformers);
+        
+        this._bindEvents();
```

Comment:
Since you only use this here, I'd either get rid of `_bindEvents` (and write it here) or make it a public method.